src/EditorFeatures/Text/Extensions.cs

```diff
@@ -64,7 +64,22 @@ public static Workspace GetWorkspace(this ITextBuffer buffer)
         /// updated to contain the same text as the snapshot if necessary. There may be multiple <see cref="Document"/>s associated with the buffer
         /// if the file is linked into multiple projects or is part of a Shared Project.
         /// </summary>
+        [Obsolete("Use GetDocuments instead")]
```

Comment:
Note: we could also not obsolete.  But i think it would be good to push people to the new method.

Reply:
Why would you not obsolete this one?

Reply:
just in case we thought that was negatively impactful on customers.